docs/dashboards/quick-start.md

```diff
@@ -1,52 +1,86 @@
-
-Any type of component can be easily added to one or more Dashboards.
+Any type of component can be easily added to one or more Dashboard reports.
 
 Once you have a dashboard created, you can start adding components to it. However, at least one or more Apps need to be created. As soon as your data is consumed and indexed by Sematext Cloud, logs, metrics, experience, events, and infrastructure components will become available to add to Dashboards. 
 
-<video style="display:block; width:100%; height:auto;" controls>
-  <source src="https://cdn.sematext.com/videos/add-components-to-dash-long.mp4" type="video/mp4" />
-</video>
+In the example below, we demonstrate the process of adding components from different solutions, selecting the preferred App to fetch data for each component, adding them to the Dashboard, and organizing the layout for improved visualization.
+
+![Add Components to Dashboard](../images/dashboards/dashboards-add-components.gif)
 
 With the custom reports features you can first create specific logging, events or metric component, and add it to your Dashboard just as you can with out-of-the-box [integration](/integration/) specific components that come included when you first create an App.
 
-The image below shows a sample of a custom Kafka report panel that was created by our DevOps team, and illustrates what will be displayed in a modal window when you add a component to a Dashboard using the components settings dropdown menu.
-
-![Add Metric Dashboards Component](https://sematext.com/docs/images/guide/dashboards/add-metric-report-to-dashboard.png "Add Metric Dashboards Component")
-
-## Available Dashboard Components
-
-- Line, Area, Bar charts metrics component binned by data creation date
-- Bar chart component binned by data creation date
-- Table of events component in reverse chronological order
-- Log table component with logs listed in reverse chronological order
-- Logs count bar chart time series component 
-- Numeric field time series component 
-- Events count bar chart time series component
-- Top N values of given log event field component
-- Pie chart
-- Doughnut chart
-- Geo map
-- 24x7 Heatmap
-- Heatbar
-- Numeric panel for a single numerical value
-- Metric panel for multiple numerical values from different Apps
-- Markdown free form editor component used to create textual panels used to add custom event notes
+The image below displays a sample of a customized Infrastructure component designed to automatically retrieve top data based on CPU, Disk, and Memory usage. To add this component, navigate to the 'Infrastructure' tab on the left and select the 'Top N Hosts' option.
+
+![Custom Infra Component](../images/dashboards/custom-infra-component.gif)
+
+## Built-in Components
+
+We provide pre-configured built-in components with predefined metrics and styles tailored to specific purposes. These components are ready to use, requiring just a click to select and add them to your dashboard.
```

Comment:
Would be good to document on the Wiki how to get/generate this list so we can regenerate it easily in the future

Reply:
I didn't generate it from somewhere, I wrote it :)

Reply:
I see....